CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 ## 3.36.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_active_directory_domain_trust` ([#2401](https://github.com/hashicorp/terraform-provider-google-beta/pull/2401))
+* **New Resource:** `google_access_context_manager_service_perimeters` ([#2382](https://github.com/hashicorp/terraform-provider-google-beta/pull/2382))
```

Comment:
This PR should have had 2 new resources `google_access_context_manager_access_levels`, sorry - i made a mistake in the mark-up, but fixed it now